apps/acceptance/lib/view.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    //Get the root element of the app and return it.
+    getRootElement: function (rootSelector) {
+        return this.client.findElement(rootSelector);
+    },
+    //From the root, find a child element based on selector.
+    findElement: function (selectorObject) {
+        return this.getRootElement(selectorObject.root).findElement(selectorObject.element);
+    },
+    //Return whether or not the root element is displayed.
+    rootDisplayed: function () {
+        return this.getRootElement().displayed();
+    },
+    //Wait for an element to become visible
+    waitForDisplay: function (selectorObject) {
+        var self = this;
+        return this.client.waitFor(function() {
+            return self.findElement(selectorObject).displayed();
+        });
+    },
+    //Wait for an element to be hidden
+    waitForHide: function () {
+        var self = this;
+        return this.client.waitFor(function () {
+            return !self.displayed();
+        });
+    },
+    //Tap on an element.
+    tap: function (selectorObject) {
+        this.actions.tap(this.findElement(selectorObject)).perform();
+        this.actions.wait(2).perform();
+    },
+    launch: function (origin) {
+        var client = this.client;
+        client.apps.launch(origin);
+        client.apps.switchToApp(origin);
```

Comment:
Like we discussed on IRC, this function would work only if we're in the System app. Let's add a `switchToFrame()`, just before, to make sure we are.